gstools/field/cond_srf.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+GStools subpackage providing a class for conditioned spatial random fields.
+
+.. currentmodule:: gstools.field.cond_srf
+
+The following classes are provided
+
+.. autosummary::
+   CondSRF
+"""
+# pylint: disable=C0103
+
+import numpy as np
+from gstools.field.generator import RandMeth
+from gstools.field.base import Field
+from gstools.krige import Krige
+
+__all__ = ["CondSRF"]
+
+GENERATOR = {
+    "RandMeth": RandMeth,
+}
```

Comment:
Maybe add a comment stating that this is the dict. of supported generators?

Reply:
Done.